### PR TITLE
start implement new settings page

### PR DESCRIPTION
**Pull request by alonbr**

### CODE REVIEW

**Comment by alonbr** [comment]
@omryYotpo Take a look at this, this is a good starting point for the new admin settings page

File: Yotpo/Yotpo/Block/Config.php [hunk 1]
```diff
@@ -10,31 +10,53 @@ class Config
...
+        \Magento\Config\Model\Resource\Config $resourceConfig
     ) {        
         $this->_scopeConfig = $scopeConfig;
+        $this->_resourceConfig = $resourceConfig;
     }
 
     public function getAppKey()
     {   
-        return $this->_scopeConfig->getValue(self::YOTPO_APP_KEY, ScopeInterface::SCOPE_STORE);     
+        return $this->_scopeConfig->getValue(self::YOTPO_APP_KEY, ScopeInterface::SCOPE_STORES, 1);
```
Comment (vladshub) [inline Yotpo/Yotpo/Block/Config.php]: @orenmagen Why did you add the 1 in every get function?

Comment (orenmagen) [inline Yotpo/Yotpo/Block/Config.php]: @vladshub 
getValue expects 3 params, we added the last one for consistency (Alon's advice)

Comment (vladshub) [inline Yotpo/Yotpo/Block/Config.php]: !completed
